EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,89 @@
+---
+eip: 7201
```

Comment:
Confirming the assignment of the EIP number as the pull request number.  Please rename the filename to match.